our-work/collaborations/fedulov.yml

```diff
@@ -1,8 +1,8 @@
 title: Fedulov Lab
 id: fedulov-lab
 description: The Fedulov lab investigates epigenetic regulation. CCV's collaboration with the Fedulov lab involved analyzing and publishing reduced-representation bisulfite sequencing data.
-tags:
-  - active
+active: True
```

Comment:
@ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?